test/versioned/amqplib/callback.test.js

```diff
@@ -355,83 +344,75 @@ tap.test('amqplib callback instrumentation', function (t) {
     let queue = null
 
     channel.assertExchange(exchange, 'direct', null, function (err) {
-      t.error(err, 'should not error asserting exchange')
+      assert.ok(!err, 'should not error asserting exchange')
 
       channel.assertQueue('', { exclusive: true }, function (err, res) {
-        t.error(err, 'should not error asserting queue')
+        assert.ok(!err, 'should not error asserting queue')
         queue = res.queue
 
         channel.bindQueue(queue, exchange, 'consume-tx-key', null, function (err) {
-          t.error(err, 'should not error binding queue')
+          assert.ok(!err, 'should not error binding queue')
+          let produceTx
+          // set up consume, this creates its own transaction
+          channel.consume(queue, function (msg) {
```

Comment:
I want to fix these up a bit more so if failures occur in the consume and end callbacks they actually throw errors and not hang the tests